app/renderer/components/common/contextMenu/contextMenuItem.js

```diff
@@ -44,6 +44,22 @@ class ContextMenuItem extends ImmutableComponent {
   get hasAccelerator () {
     return this.accelerator !== null
   }
+
+  getYAxis (e) {
```

Comment:
this could definitely use a small description; I know you didn't write it (thanks for pulling out to a function 😄 ), but it's a little tricky to follow

Reply:
done